### PR TITLE
Add landing page

### LINKED ISSUE #2: Explain what's transport.beta.gouv.fr and why it's cool
When I visit transport.beta.gouv.fr
    I want to know what they do,
    so that I can be reassured I'm in the right place,
    and build my shiny app around open transport data.

### PR DESCRIPTION
**Pull request by maukoquiroga**
Fixes #2 

capture d ecran 2017-09-07 a 10 25 44

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
-assets/node_modules
-assets/elm-stuff
+client/node_modules
```
Comment (Morendil) [inline .eslintignore]: Maybe use a single file for eslintignore and jshintignore ? This is recommended in https://eslint.org/docs/user-guide/configuring#using-an-alternate-file

File: client/javascripts/Transport.elm [hunk 2]
```diff
@@ -0,0 +1,35 @@
+module Transport exposing (..)
+
+import Html exposing (..)
+
+-- MAIN
+
+main : Html String
```
Comment (Morendil) [inline client/javascripts/Transport.elm]: Extracting `view` from `main` without going straight to [TEA](https://guide.elm-lang.org/architecture/) strikes me as a somewhat odd choice; I'd be interested to know your reasoning. I've tended to start with `Html.beginnerProgram` as the next step up from a bare `main`.

File: client/tests/TransportSpec.elm [hunk 3]
```diff
@@ -0,0 +1,27 @@
+module TransportSpec exposing (..)
+
+import Test exposing (..)
+import Test.Html.Query as Query
```
Comment (Morendil) [inline client/tests/TransportSpec.elm]: Oo, I didn't know about that package. Looks like a good starting point as far as unit tests are concerned. Looking forward to seeing what comes next. :)

File: client/javascripts/Transport.elm [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+-- VIEW
+
+view : Html String
+view =
+    section []
+    [ h1 [] [ text "transport.beta.gouv.fr" ]
+    , h2 [] [ text "Rendre disponible, valoriser et améliorer les données transports" ]
+    , h3 [] [ text "transport.beta.gouv.fr c'est quoi ?" ]
+    , p  [] [ text "C'est la plateforme de diffusion des données ouvertes relatives au transport. Mais c'est aussi la plateforme qui permet à la communauté des diffuseurs et des ré-utilisateurs de se rassembler, de dialoguer, de réfléchir ensemble aux problèmes autour des données transport (qualité, complétude, licences) et de construire les outils pour les résoudre." ]
+    , h3 [] [ text "Pourquoi transport.beta.gouv.fr ?" ]
```
Comment (Morendil) [inline client/javascripts/Transport.elm]: I might be tempted to *start* refactoring to let latent structure appear. It's a good thing not to push it too far in the early stages of an app, lest you end up with premature abstractions, but that's quite a large function. But it's hard to tell how to handle that without knowing more about where you're heading next.

File: client/javascripts/Transport.elm (hunk 2 above)
Comment (maukoquiroga) [reply client/javascripts/Transport.elm]: Yep, I'm prototyping. It'll certainly evolve into `Ports`, `Pages` and `Models` or `Data` for the type aliases.

File: client/javascripts/Transport.elm (hunk 4 above)
Comment (maukoquiroga) [reply client/javascripts/Transport.elm]: @Morendil Thing is I don't yet how much of that content will make it to the landing page. I'll greatly appreciate a text review as well. I haven't had the time to enable review apps yet, sadly, but running the app locally should be trivial if you follow the README.

File: client/tests/TransportSpec.elm (hunk 3 above)
Comment (maukoquiroga) [reply client/tests/TransportSpec.elm]: I'm trying to hack my way into behaviour-driven elm, which ATM seems to be a rather lonely path. Check this out as well: https://github.com/brian-watkins/elmer

File: README.md [hunk 5]
```diff
@@ -2,15 +2,16 @@
...
   * Install Node.js dependencies with `mix npm install`
   * Install Elm dependencies with `mix elm install`
-  * Optional: install linters with `npm install -g eslint jshint sass-lint elm`
+  * Optional: install linters with `npm install -g eslint sass-lint elm`
 
 Usage:
+
   * Run the up with `mix phx.server`
-  * Run the test with `mix test`
+  * Run the specs with `mix test` and `mix elm test`
```
Comment (l-vincent-l) [inline README.md]: I will prefer to have consistency in names.
We should use test or specs, but not both

**Comment by maukoquiroga** [comment]
@l-vincent-l RTM ?

File: README.md [hunk 6]
```diff
@@ -10,7 +10,7 @@ Installation:
 
 Usage:
   * Run the up with `mix phx.server`
-  * Run the test with `mix test`
+  * Run the specs with `mix test` and `mix elm test`
```
[the remark this answers is not in the document]
Comment (maukoquiroga) [reply README.md]: Yep, changed it thanks.

File: README.md (hunk 5 above)
Comment (maukoquiroga) [reply README.md]: Done

File: lib/mix/tasks/elm.ex [hunk 7]
```diff
@@ -5,10 +5,30 @@ defmodule Mix.Tasks.Elm do
...
+    elm_install_app()
+    elm_install_test()
+  end
+
+  def run(["test" | _tail]) do
+    case Mix.shell.cmd("cd ./client && #{@elm_test_path}", stderr_to_stdout: true) do
+      0 -> :ok
+      a -> raise "elm command failure exit code: #{a}"
+    end
+  end
```
[the remark this answers is not in the document]
Comment (maukoquiroga) [reply lib/mix/tasks/elm.ex]: But in different modules, which gives you ```mix elm blabla``` and ```mix npm blabla```

File: client/javascripts/Transport.elm [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+    , p  [] [ text "Les données de transport lorsqu’elles existent, sont exposées sur différents portails de diffusion de données ouvertes. On constate souvent un manque d’homogénéité entre les différentes sources ce qui limite la réutilisation des données et freine le déploiement de solutions innovantes sur tout le territoire ou pour tous les utilisateurs. transport.beta.gouv.fr a pour but de résoudre ces différents problèmes et de maximiser les réutilisations pour permettre :" ]
+    , ul []
+        [ li [] [ text "aux usagers des transports de mieux connaitre l'offre et ainsi de mieux préparer leurs différents déplacements, " ]
+        , li [] [ text "que soient expérimentés et proposés de nouveaux services de mobilités pour tous les usagers sur le territoire national mais aussi au niveau européen grâce à la mise en place de portails similaires dans d'autres pays." ]
+        ]
+    , h3 [] [ text "Comment fonctionne transport.beta.gouv.fr ?" ]
+    , p  [] [ text "Pour accéder/en savoir plus sur les  jeux de données qui sont exposés sur transport.beta.gouv.fr consultez cette page : http://asdf1234.fr" ]
```
[the remark this answers is not in the document]
Comment (maukoquiroga) [reply client/javascripts/Transport.elm]: I'll definitely do it in a later PR, when I actually structure ELM code properly. For now it is just HTML. If that's ok for you.

**Pull request merged**